src/common/Utilities/Util.h

```diff
@@ -765,7 +761,10 @@ class EventMap
                 else if (_phase && (itr->second & 0xFF000000) && !(itr->second & (_phase << 24)))
                     _eventMap.erase(itr);
```

Comment:
It'd be kind if you could enclose this in brackets too :D

Reply:
its not part of my PR code but fine